.github/workflows/build-test.yml

```diff
@@ -196,7 +196,7 @@ jobs:
       - name: Install rust
         uses: actions-rs/toolchain@v1
         with:
-            toolchain: nightly
+            toolchain: stable
```

Comment:
Is this intentional? I'm not sure what the rational was for using `nightly` for `fmt` and `clippy`, but it does seem like using nightly could be a little chaotic.

Reply:
yes, CI was failing last night. Specifically, it was emitting errors for `num-bigint`